conda_metadata_app/pages/search_by_file_path_page.py

```diff
@@ -49,7 +49,7 @@ def find_artifacts_by_path(path):
     initial_sidebar_state="collapsed",
 )
 
-if not app_config().enable_filepath_search:
+if not AppConfig().enable_filepath_search:
     st.error("File path search is disabled in the app configuration.")
     st.stop()
```

Comment:
I don't think we need this anymore. It can be instead hidden dynamically in `app.py`.

Reply:
b0561947e0fe9f17b8a9074d9a7e9ba84dba69d1

Reply:
I very much like this! Good catch!